hack/test/osctl-cluster-create.sh

```diff
@@ -53,6 +53,9 @@ run "timeout=\$((\$(date +%s) + ${TIMEOUT}))
 	   sleep 5
 	 done"
 
+## Give a moment for things to stabilize before applying manifests
+sleep 10
```

Comment:
Not for this PR, but curious what we can wait on via `kubectl wait` before we apply this. Only started seeing this after 1.16 move.